archive.php

```diff
@@ -1,40 +1,43 @@
 <?php
+
 /**
  * The template for displaying Archive pages.
  *
  * Used to display archive-type pages if nothing more specific matches a query.
  * For example, puts together date-based pages if no date.php file exists.
  *
- * Learn more: http://codex.wordpress.org/Template_Hierarchy
- *
- * Methods for TimberHelper can be found in the /lib sub-directory
+ * Learn more: https://developer.wordpress.org/themes/basics/template-hierarchy/
  *
- * @package  WordPress
- * @subpackage  Timber
- * @since   Timber 0.2
  */
 
-$templates = array( 'archive.twig', 'index.twig' );
+namespace Timber\StarterTheme;
+
+use Timber\Timber;
+
+$templates = array('archive.twig', 'index.twig');
 
 $context = Timber::context();
```

Comment:
I think now, context is not passed to `Timber::render()` anymore.

Reply:
Do'h! Mixed up the render and context methods.. Fixed now! Also removed the $post here since that will probably also be provided. Will test this as well later on this week.